airbyte-integrations/connectors/source-scaffold-cdk/README.md

```diff
@@ -0,0 +1,98 @@
+# Scaffold Cdk Source
```

Comment:
One thing I'm not sure about is the most logical place to provide an entrypoint for TODOs/summary of what is happening. I went back and forth with the `README.md` and the `source.py` file and settled on the latter since it's clear this README is geared toward after the source has already been built.

One other idea is to output a separate top-level README file (e.g. README-DEVELOPMENT-GUIDE.md or a better name) explaining the layout of the project and where a dev should start poking at. 

What do you think?

Reply:
I'm in favor of scattering todos throughout the module. This way you can grep the entire directory for TODO and you will find what needs to be done.

Reply:
I should have been clearer. I was suggesting something else in addition to scattered todos. This 'something' would:
1) tell people to look for todos
2) briefly explain the repo layout for those curious
3) if we think this is a good idea, can either be a separate README, a section within the current README or part of the source.py comments.

Reply:
https://github.com/airbytehq/airbyte/pull/3043/files#diff-3dcbe90c0f1e986b5b5aeb2b1c192e318e14ffcc853aa2bf4411c3f8b582434aR1 is an example to what I was thinking of, except at the top-level.